packages/vm/lib/evm/opcodes/EIP2929.ts

```diff
@@ -0,0 +1,102 @@
+import BN = require('bn.js')
+import { Address } from 'ethereumjs-util'
+import { RunState } from './../interpreter'
+import { addressToBuffer } from './util'
+
+/**
+ * Adds address to accessedAddresses set if not already included.
+ * Adjusts cost incurred for executing opcode based on whether address read
+ * is warm/cold. (EIP 2929)
+ * @param {RunState} runState
+ * @param {BN}       address
+ */
+export function accessAddressEIP2929(runState: RunState, address: Address, baseFee?: number) {
+  if (!runState._common.eips().includes(2929)) return
```

Comment:
Just seeing here that this implementation has also these `common.eips().includes()` calls. We should really replace this soon by a method in common also taking the HFs into account. This will otherwise fall onto our feet.

Reply:
(so "soon" means: in the next 1-2 weeks)

Reply:
I wanted to introduce this in the EIP2718 PR, but won't do it as it will get too bloated otherwise. Can you open an issue to is so we don't "forget" it? 😄

Reply:
I will just prioritize this as the first task I'll do on Monday, so we won't forget then. 😄